src/app/user/(logged-in)/my-pets/dogs/[dogId]/edit/page.tsx

```diff
@@ -21,7 +21,8 @@ export default async function Page(props: { params: { dogId: string } }) {
     actor,
     dogId,
   );
-  if (error) {
+  if (error !== undefined) {
+    // TODO: show some error on the page?
```

Comment:
make ticket

Reply:
no need for a ticket. just show a simple error message and ask the user to reload their page. - dfb3795